### PR TITLE
added Digi clock by cpp

### PR DESCRIPTION
**Pull request by Nayan808**
Hello Project Admin,
### I have added a Digital Clock made With Cpp which I mention in the issue no. #2671 .

It can Also Used To Set Alarm 

## Screenshot -
Screenshot from 2023-06-25 20-54-01

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### It is super ready to `Merge`

### CODE REVIEW

File: Drum-Sounds/.DS_Store
Comment (MohitGupta121) [inline Drum-Sounds/.DS_Store:1]: @Nayan808 remove this file from PR

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@Nayan808 remove all the images from PR and upload them somewhere else and then use there link to fetch the images in your project image src instead of direct image path

File: Drum-Sounds/.DS_Store
Comment (Nayan808) [reply Drum-Sounds/.DS_Store:1]: wait i will raise another pr request with alll this conflicts to be done ok

**Comment by MohitGupta121** [comment]
Okay @Nayan808 then I closing this one.

**Comment by Nayan808** [comment]
Hey @MohitGupta121 listen I had raised # PR can  u Please merge the other 2 as i was working on drum sound project for while i need some time for that please can you merge the other two pr #2670 #2672

**Pull request closed without merging**